node/container.go

```diff
@@ -65,6 +87,7 @@ func (a *HostAgent) StopContainer(serviceID string, instanceID int) error {
 		return err
```

Comment:
The event monitor receives an event when the container exits and will update the state accordingly